members/CAL000350.yaml

```diff
@@ -38,7 +38,7 @@ contact_form:
     - javascript:
         - name: accent-replace
           values: ["$NAME_FIRST", "$NAME_LAST"]
-          selectors: ["#txtFirstName", "#txtLastName"]
+          selectors: ["#txtFirstName", "#txtLastName", "#txtAddress"]
```

Comment:
You didn't add the address to `values` above

Reply:
added